ui/src/components/core/button/button.stories.tsx

```diff
@@ -24,14 +24,14 @@ export const Default = () => (
     <Button
       colorScheme="blue"
       variant="outline"
-      css={{ py: '$1', borderRadius: '$md' }}
+      css={{ py: '$1', height: '$5', borderRadius: '$md' }}

```

Comment:
Seems like change the css in the stories will make the component look different on storybook than in the project usage, is that correct?

Reply:
Yeap, cause this is a md button, but with some specific changes, like [this one](https://www.figma.com/file/98aKyXDUKu83UoFwGy5BGV/NFA-App?node-id=2%3A10577&t=6Sea988a3GTnRqIb-0), so I did this on the storybook to check how it was being renderer, but then on the view the button would look like this (I mean, the code)